<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlopsConfigWidget</class>
 <widget class="QWidget" name="FlopsConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>577</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>774</width>
        <height>507</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Data Limits</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="dataGroupBox">
         <property name="title">
          <string>Data</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="startDataLabel">
            <property name="text">
             <string>Start at:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="dataStartAtBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000000000</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="maxDatalabel">
            <property name="text">
             <string>Maximum:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="dataMaxDataBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000000000</number>
            </property>
            <property name="value">
             <number>100000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="workSizeGroupBox">
         <property name="title">
          <string>Workgroup Size</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="amountOfDataLabel">
            <property name="text">
             <string>Amount of data:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="workGroupDataBox">
            <property name="maximum">
             <number>1000000</number>
            </property>
            <property name="value">
             <number>100000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>774</width>
        <height>507</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Select Benchmark</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="addButton">
         <property name="text">
          <string>Addition</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mulButton">
         <property name="text">
          <string>Multiply</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="divButton">
         <property name="text">
          <string>Divide</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="madButton">
         <property name="text">
          <string>Multiply Add (MAD)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
